unite_contranges.py

```diff
@@ -0,0 +1,146 @@
+#Allison Towner
+#09 October 2020
+
+#continuum_selections.py in analysis code is located at:
+#in ALMA_IMF/reduction/analysis/
+
+def merge_contdotdat(field='',band=''):
+
+	import numpy as np
+	import sys
+	sys.path.append('.')
+	from parse_contdotdat import parse_contdotdat
```

Comment:
this should generally go at the top (you want to do imports at import time, usually)